framework/src/main/java/org/checkerframework/common/reflection/DefaultReflectionResolver.java

```diff
@@ -506,7 +506,6 @@ private boolean isUnknownMethod(MethodInvocationTree tree) {
      * @param env the environment
      * @return the (potentially empty) set of corresponding method Symbol(s)
      */
-    @SuppressWarnings("interning:not.interned") // bug?
```

Comment:
The errors in this method are because `com.sun.tools.javac.util.Name` is used. `com.sun.tools.javac.util.Name` implements  `javax.lang.model.element.Name`, so I think we can also annotate it as `@Interned`.